packages/react-storage/src/components/StorageBrowser/adapters/permissionParsers.ts

```diff
@@ -0,0 +1,68 @@
+import { LocationPermissions } from '../actions/handlers/types';
+import { Permission, StorageAccess } from '../storage-internal';
+
+export const parseAccessGrantPermission = (
```

Comment:
do we need separate unit test on individual functions to boost coverage here? or does existing tests cover all branches already here?

Reply:
Here's the dedicated tests: https://github.com/aws-amplify/amplify-ui/pull/6073/files#diff-e300cdf4afb08cd85469f1eb888f3f44210ff3bcdf05bec2f2aac158d9ff4524